js/fizzbuzz.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+var FizzBuzz = (function() {
+
+  var array = [];
+  var fizzBuzz = function (firstWord, secondWord) {
+
+    if (firstWord !== '') {
```

Comment:
This could be simplified to `if (!firstWord) { /* ... */} -- if it's not defined as an argument, it'll be `undefined`, which is a "falsy" value.